README.md

```diff
@@ -36,23 +36,42 @@ Read about supported commands in the [CLI documentation website](https://develop
 Usage:
   flow [command]
 
-Available Commands:
-  accounts     Utilities to manage accounts
-  blocks       Utilities to read blocks
+👋 Welcome Flow developer!
+   If you are starting a new flow project use our super commands, start by running 'flow setup'. 
+
+🔥 Super Commands
```

Comment:
should we include any required or option args in this list?

Reply:
Not necessary for this. Take a look at the home readme now. It's just out of date. Can drill into the commands for more details.